BurnOutSharp/PackerType/PECompact.cs

```diff
@@ -8,9 +11,49 @@ public string CheckContents(string file, byte[] fileContent, bool includePositio
             // "PEC2"
             byte[] check = new byte[] { 0x50, 0x45, 0x43, 0x32 };
             if (fileContent.FirstPosition(check, out int position, end: 2048))
+            {
+                string version = GetVersion(file, position);
+                if (version == null)
+                    return "PE Compact 2" + (includePosition ? $" (Index {position})" : string.Empty);
+                return $"PE Compact 2 Version {version}" + (includePosition ? $" (Index {position})" : string.Empty);
```

Comment:
Nitpick: Please add a newline after the if block here and in the version method.